src/components/application/StatsResults.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from "react";
+
+export default function({ body }) {
+
+    const [ user, setUser ] = useState("");
+
+    function ForUser({ user }) {
```

Comment:
I'm not sure what this inner component is doing here, it looks like unreachable code?

For consistency, components should have their own files and folder for contains script/style/tests/etc, rather than be defined inside something else.

Reply:
Ah yes, I hadn't quite gotten around to fully hooking this up - I have removed for now.